mp/src/game/shared/da/da_player_shared.cpp

```diff
@@ -799,7 +799,7 @@ void CDAPlayerShared::StartSliding(bool bDiveSliding)
 void CDAPlayerShared::EndSlide()
 {
 	// If it was long enough to notice what it was, then train the slide.
-	if (gpGlobals->curtime - m_flSlideTime > 1)
+	if (m_pOuter->GetCurrentTime() - m_flSlideTime > 1)
```

Comment:
The problem is that m_flSlideTime is assigned from m_pOuter->GetCurrentTime() here:
https://github.com/BSVino/DoubleAction/blob/develop/mp/src/game/shared/da/da_player_shared.cpp#L795

I don't know how you implemented ``m_pOuter->GetCurrentTime()``, but if it is to be slowmo-adjusted and ``gpGlobals->curtime`` is not, the two will diverge.

The effect would be that ``m_pOuter->GetCurrentTime()`` is always less than or equal to ``gpGlobals->curtime``.
And usually it will be more than a second behind, which means that this if will usually hold true, regardless of how long the person was sliding.